From: Mike Gilbert <floppym@gentoo.org>
Date: Fri, 30 Mar 2018 11:00:17 -0400
Subject: units: initctl: move the fifo to /run/initctl to match sysvinit

The fifo location was moved in sysvinit-2.89.

http://git.savannah.nongnu.org/cgit/sysvinit.git/commit/?id=80dbcf3de3c1b83aeaa713a8fe5b8d35d8649af2
(cherry picked from commit 4d8c7c1b3a5feebca948a3b8663f5be887b57731)
---
 units/systemd-initctl.service.in | 2 +-
 units/systemd-initctl.socket     | 4 ++--
 2 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/units/systemd-initctl.service.in b/units/systemd-initctl.service.in
index 6cfed3d..2b4b957 100644
--- a/units/systemd-initctl.service.in
+++ b/units/systemd-initctl.service.in
@@ -8,7 +8,7 @@
 #  (at your option) any later version.
 
 [Unit]
-Description=/dev/initctl Compatibility Daemon
+Description=initctl Compatibility Daemon
 Documentation=man:systemd-initctl.service(8)
 DefaultDependencies=no
 
diff --git a/units/systemd-initctl.socket b/units/systemd-initctl.socket
index 61f877b..9d97579 100644
--- a/units/systemd-initctl.socket
+++ b/units/systemd-initctl.socket
@@ -8,12 +8,12 @@
 #  (at your option) any later version.
 
 [Unit]
-Description=/dev/initctl Compatibility Named Pipe
+Description=initctl Compatibility Named Pipe
 Documentation=man:systemd-initctl.service(8)
 DefaultDependencies=no
 Before=sockets.target
 
 [Socket]
-ListenFIFO=/run/systemd/initctl/fifo
+ListenFIFO=/run/initctl
 Symlinks=/dev/initctl
 SocketMode=0600
